aws/lambda-transform/massdriver.yaml

```diff
@@ -0,0 +1,142 @@
+schema: draft-07
+name: lambda-transform
+description: reads data from sqs, performs a data transform, and writes the transformed data to the next topic.
+source_url: github.com/massdriver_cloud/application_examples/aws/lambda-transform
+access: private
+type: application
+
+params:
+  required:
+    - region
+  properties:
+    region:
+      title: AWS Region 
+      $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/aws-region.json
+    application:
+      type: object
+      required: 
+        - addend
+      properties:
+        addend:
```

Comment:
This and the title are spelled differently. Not sure which one is right since it looks like a make-believe word.

Reply:
The numbers or the objects to be added in general addition are collectively referred to as the terms,[[6]](https://en.wikipedia.org/wiki/Addition#cite_note-6) the addends[[7]](https://en.wikipedia.org/wiki/Addition#cite_note-Shmerko-7)[[8]](https://en.wikipedia.org/wiki/Addition#cite_note-Schmid_1974-8)[[9]](https://en.wikipedia.org/wiki/Addition#cite_note-:1-9) or the summands;[[10]](https://en.wikipedia.org/wiki/Addition#cite_note-10) this terminology carries over to the summation of multiple terms. This is to be distinguished from factors, which are [multiplied](https://en.wikipedia.org/wiki/Multiplication). Some authors call the first addend the augend.[[7]](https://en.wikipedia.org/wiki/Addition#cite_note-Shmerko-7)[[8]](https://en.wikipedia.org/wiki/Addition#cite_note-Schmid_1974-8)[[9]](https://en.wikipedia.org/wiki/Addition#cite_note-:1-9) In fact, during the [Renaissance](https://en.wikipedia.org/wiki/Renaissance), many authors did not consider the first addend an "addend" at all. Today, due to the [commutative property](https://en.wikipedia.org/wiki/Commutative_property) of addition, "augend" is rarely used, and both terms are generally called addends.[[11]](https://en.wikipedia.org/wiki/Addition#cite_note-Schwartzman_p._19-11)

Reply:
I feel like I learned an area of maths I never knew. Big fan of `addends` 🤞